src/commons/BrowserUtils.ts

```diff
@@ -352,11 +352,12 @@ export namespace BrowserUtils {
     isExist(selector);
 
     Reporter.debug(`Switching to an Iframe by selector '${selector}'`);
+    chillOut();
```

Comment:
chillout if needed should be after switch

Reply:
moved to after switch